### PR TITLE
fix: Update gateway setting conflect

### PR DESCRIPTION
**Pull request by weili520**
Signed-off-by: TheYoungManLi <cjl@kubesphere.io>

### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes:
Fixes ##2723

### Special notes for reviewers:

https://user-images.githubusercontent.com/33231138/144956525-c52ec171-5725-4013-9e8e-c938895ce56d.mov

### Does this PR introduced a user-facing change?
```release-note
NONE
```

### CODE REVIEW

File: locales/en/l10n-projects-projectSettings-gatewaySettings-list.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ module.exports = {
   // Project Gateway > Enable Gateway > NodePort
   // Project Gateway > Enable Gateway > LoadBalancer
   LOAD_BALANCER_PROVIDER: 'Load Balancer Provider',
+  GATEWAY_UPDATING_TIP: 'Gateway is updating, please try again later.',
```
Comment (Felixnoo) [inline locales/en/l10n-projects-projectSettings-gatewaySettings-list.js:32]: `Gateway is updating, please try again later.` -> `Updating the gateway. Please try again later.`

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Hi, please see my comment. Thanks.

File: locales/en/l10n-projects-projectSettings-gatewaySettings-list.js (hunk 1 above)
Comment (weili520) [reply locales/en/l10n-projects-projectSettings-gatewaySettings-list.js]: ok, thanks

**Pull request merged**